src/renderer/components/chat/ChatListItemRow.tsx

```diff
@@ -39,9 +41,13 @@ export const ChatListItemRowChat = React.memo<{
         onClick={onChatClick.bind(null, chatId)}
         onContextMenu={event => {
           const chat = chatCache[chatId]
-          if (chat?.type === 'ChatListItem')
+          if (chat?.type === 'ChatListItem') {
+            setHoverChatListItem(true)
             openContextMenu(event, chat, selectedChatId)
+            setHoverChatListItem(false)
```

Comment:
while this could work in theory, `openContextMenu` is not blocking (which would be bad, because blocking would mean that the UI would be frozen ;). This would be a case for async, like `await openContextMenu`, but for that to work the context menu needs to be patched to return a promise when it was clicked away, closed, or once an action was triggered.

My first idea was building in the promise into `useChatListContextMenu`, but that would be really ugly / hard to maintain code, like resolving the promise in each action callback, so I think it would make more sense to build it directly into `openContextMenu`. So change `openContextMenu`  to return a promise that gets resolved in all cases where it is closed (also when an action was chosen it is also closed, so maybe in the function that closes it? but I haven't looked at the code in a while so take it with a grain of salt.).

Reply:
oh sorry I thought it was working but it seems I was mistaken.